DigitalLearningSolutions.Data.Tests/Services/CourseServiceTests.cs

```diff
@@ -0,0 +1,95 @@
+﻿namespace DigitalLearningSolutions.Data.Tests.Services
+{
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Transactions;
+    using DigitalLearningSolutions.Data.DataServices;
+    using DigitalLearningSolutions.Data.Models.Courses;
+    using DigitalLearningSolutions.Data.Services;
+    using FakeItEasy;
+    using FluentAssertions;
+    using NUnit.Framework;
+
+    public class CourseServiceTests
+    {
+        private const int CentreId = 2;
+        private const int AdminCategoryId = 0;
+        private ICourseDataService courseDataService = null!;
+        private CourseService courseService = null!;
+
+        [SetUp]
+        public void Setup()
+        {
+            courseDataService = A.Fake<ICourseDataService>();
+            A.CallTo(() => courseDataService.GetCourseStatisticsAtCentreForCategoryId(CentreId, AdminCategoryId))
+                .Returns(GetSampleCourses());
+
+            courseService = new CourseService(courseDataService);
+        }
+
+        [Test]
+        public void GetTopCourseStatistics_should_return_correctly_ordered_active_course_statistics()

```

Comment:
Could we specify here: "ordered by In progress field" or something similar to make "Correctly" unambiguous.